spanner/google/cloud/spanner_v1/database.py

```diff
@@ -416,9 +416,10 @@ def run_in_transaction(self, func, *args, **kw):
         :param args: additional positional arguments to be passed to ``func``.
 
         :type kw: dict
-        :param kw: optional keyword arguments to be passed to ``func``.
+        :param kw: (Optional) keyword arguments to be passed to ``func``.
                    If passed, "timeout_secs" will be removed and used to
-                   override the default timeout.
+                   override the default timeout which defines maximum timestamp
```

Comment:
Perhaps reword it to `override the retry timeout` as `default timeout` is still vague and not particular helpful.

Reply:
I got your point and I think it better be called `the default retry timeout`.

Reply:
SGTM